### PR TITLE
Added Cygwin 3.3.5

### PR DESCRIPTION
**Pull request by licon4812**
- [ yes ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ yes ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ yes ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ yes ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ yes ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/63513)

### CODE REVIEW

File: manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Cygwin.Cygwin
+PackageVersion: 3.3.4
```
Comment (ItzLevvie) [inline manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.installer.yaml:5]:
```suggestion
PackageVersion: 3.3.5
```
Output shows that it installs 3.3.5? You will also have to update the package version folder from `3.3.4` to `3.3.5`
```
Augmented Transaction List:
   [...]
   1 install cygwin          3.3.5-1
   [...]
```

File: manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Cygwin.Cygwin
+PackageVersion: 3.3.4
```
Comment (ItzLevvie) [inline manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.3.5
```
Output shows that it installs 3.3.5? You will also have to update the package version folder from `3.3.4` to `3.3.5`
```
Augmented Transaction List:
   [...]
   1 install cygwin          3.3.5-1
   [...]
```

File: manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Cygwin.Cygwin
+PackageVersion: 3.3.4
```
Comment (ItzLevvie) [inline manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.yaml:5]:
```suggestion
PackageVersion: 3.3.5
```
Output shows that it installs 3.3.5? You will also have to update the package version folder from `3.3.4` to `3.3.5`
```
Augmented Transaction List:
   [...]
   1 install cygwin          3.3.5-1
   [...]
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by licon4812** [comment]
@ItzLevvie  since this download location changes when a new version is released would we be able to get the update process done automatically whenever they change the installer at that location?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**